### ISSUE TITLE
init_from fails

### ISSUE THREAD

**Issue by stephenpardy** [jcjohnson/torch-rnn#40]
Ran the training for a while before quitting it with ctrl+c. Restarting throws this error:

>>> th train.lua -init_from cv/checkpoint_80000.t7 -gpu -1
Running in CPU mode	
Initializing from 	cv/checkpoint_7000.t7	
Epoch 1.00 / 50, i = 1 / 17800, loss = 12.962780	
Epoch 1.01 / 50, i = 2 / 17800, loss = 11.756465	
Epoch 1.01 / 50, i = 3 / 17800, loss = 11.562723	
Epoch 1.01 / 50, i = 4 / 17800, loss = 11.154325	
Epoch 1.01 / 50, i = 5 / 17800, loss = 10.792833	
Epoch 1.02 / 50, i = 6 / 17800, loss = 10.426821	
Epoch 1.02 / 50, i = 7 / 17800, loss = 10.035604	
Epoch 1.02 / 50, i = 8 / 17800, loss = 9.824880	
Epoch 1.03 / 50, i = 9 / 17800, loss = 9.523673	
Epoch 1.03 / 50, i = 10 / 17800, loss = 9.234539	
Epoch 1.03 / 50, i = 11 / 17800, loss = 9.018742	
Epoch 1.03 / 50, i = 12 / 17800, loss = 8.913129	
Epoch 1.04 / 50, i = 13 / 17800, loss = 8.635086	
/Users/spardy/torch/install/bin/luajit: /Users/spardy/torch/install/share/lua/5.1/nn/Container.lua:69: 
In 1 module of nn.Sequential:
...rs/spardy/torch/install/share/lua/5.1/nn/LookupTable.lua:62: index out of range at /Users/spardy/torch/pkg/torch/lib/TH/generic/THTensorMath.c:156
stack traceback:
	[C]: in function 'index'
	...rs/spardy/torch/install/share/lua/5.1/nn/LookupTable.lua:62: in function <...rs/spardy/torch/install/share/lua/5.1/nn/LookupTable.lua:56>
	[C]: in function 'xpcall'
	/Users/spardy/torch/install/share/lua/5.1/nn/Container.lua:65: in function 'rethrowErrors'
	/Users/spardy/torch/install/share/lua/5.1/nn/Sequential.lua:44: in function 'forward'
	train.lua:124: in function 'opfunc'
	/Users/spardy/torch/install/share/lua/5.1/optim/adam.lua:33: in function 'adam'
	train.lua:181: in main chunk
	[C]: in function 'dofile'
	...ardy/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:145: in main chunk
	[C]: at 0x010f1f3d50

I get the same error, at the same index no matter what restart file I choose. Restarting the training from scratch does not give any errors and I am able to sample from any of the restart files without error.

**Comment by stephenpardy** [comment]
Just tried with the sample text file (tiny-shakespeare.txt) and it runs and restarts without error. Are there any gotchas in the text formatting that I should check?

**Comment by stephenpardy** [comment]
Found that there was a mistake with my input file where many sections of text were repeated over and over. When I fixed this the problem went away. I am just going to assume this was some user error so I will close this.
